exercises/practice/saddle-points/saddle-points.jl

```diff
@@ -0,0 +1,2 @@
+function saddlepoints(matrix)
```

Comment:
```suggestion
function saddlepoints(M)
```

If renaming as Sascha suggested.